tools/firtool/firtool.cpp

```diff
@@ -42,6 +43,9 @@
 #include "llvm/Support/SourceMgr.h"
 #include "llvm/Support/ToolOutputFile.h"
 
+#include <chrono>
```

Comment:
MLIR and LLVM have wrappers around `chrono` that provides `TimePoint`. This is likely better to use here than `std::chrono` directly.

Reply:
Thanks, I changed. But it seems to be necessary to use `std::chrono::durartion` and `std::chrono::seconds` directly.